app/src/androidTest/kotlin/explorer/city/com/cityexplorer/ExampleInstrumentedTestKotlin.kt

```diff
@@ -0,0 +1,39 @@
+package explorer.city.com.cityexplorer
+
+import android.support.test.espresso.Espresso.onData
+import android.support.test.espresso.Espresso.onView
+import android.support.test.espresso.assertion.ViewAssertions.matches
+import android.support.test.espresso.intent.rule.IntentsTestRule
+import android.support.test.espresso.matcher.ViewMatchers.isDisplayed
+import android.support.test.espresso.matcher.ViewMatchers.withId
+import android.support.test.filters.LargeTest
+import android.support.test.rule.ActivityTestRule
+import android.support.test.runner.AndroidJUnit4
+import explorer.city.com.cityexplorer.view.activity.MainActivity
+import org.hamcrest.core.IsAnything.anything
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+
+
+/**
+ * Instrumentation test, which will execute on an Android device.
+ *
+ * @see [Testing documentation](http://d.android.com/tools/testing)
+ */
+
+@LargeTest
```

Comment:
This is deprecated and should no longer be used. More info: https://developer.android.com/reference/android/test/suitebuilder/annotation/LargeTest.html